src/ORM/Connect/PDOConnector.php

```diff
@@ -64,6 +64,18 @@ class PDOConnector extends DBConnector
      */
     protected $cachedStatements = array();
 
+    /**
+     * Driver
+     * @var string
+     */
+    private $driver = null;
```

Comment:
I think we should use protected over private

Reply:
I don't necessarily disagree, although I should point out a previous discussion where you suggested that protected variables are part of our semver-governed "Public API". I'm worried about doing both.

I think that we should either treat protected properties that aren't explicitly documented as public API as "internal", or we use privates more often.

Reply:
Yeah I guess it's a topic for another conversation, but at the moment our codebase is almost all protected over private so I guess we should keep it consistent?